library/sources/Hono.test.ts

```diff
@@ -506,3 +517,56 @@ t.test("invalid json body", opts, async (t) => {
   t.same(response.status, 400);
   t.same(await response.text(), "Invalid JSON");
 });
+
+t.test("test access only allowed for some IP addresses", opts, async (t) => {
+  // Update the allowed IP addresses
+  shouldReturnOnlyAllowedIPAddresses = true;
+  await getInstance()!.updateBlockedLists();
```

Comment:
Any way to avoid accessing a private method? Can we use the fake timer?

Reply:
Yes, made a separate test file